.devops/deploy-pipelines.yml

```diff
@@ -124,49 +124,25 @@ stages:
                 !CODEOWNERS
                 !__*/**/*
             displayName: 'Copy deploy files'
```

Comment:
You need to publish the artifact, otherwise files get lost with the job

Reply:
right, my bad.. I fixed it